business/istio_config.go

```diff
@@ -278,6 +278,11 @@ func (in *IstioConfigService) UpdateIstioConfigDetail(api, namespace, resourceTy
 	promtimer := internalmetrics.GetGoFunctionMetric("business", "IstioConfigService", "UpdateIstioConfigDetail")
 	defer promtimer.ObserveNow(&err)
 
+	return in.ModifyIstioConfigDetail(api, namespace, resourceType, resourceSubtype, name, jsonPatch, in.k8s.UpdateIstioObject)
+}
+
+func (in *IstioConfigService) ModifyIstioConfigDetail(api, namespace, resourceType, resourceSubtype, name, json string, fn func(string, string, string, string, string) (kubernetes.IstioObject, error)) (models.IstioConfigDetails, error) {
```

Comment:
Is 'ModifyIstioConfigDetail' intended to be used outside of business ? Otherwise it sounds more like a helper, I'd put it in lowercase.